src/seedsigner/gui/screens/seed_screens.py

```diff
@@ -427,6 +427,21 @@ def __post_init__(self):
         )
         self.components.append(self.fingerprint_icontl)
 
+class SeedSwitchElectrumModeScreen(ButtonListScreen):
+    button_data: list = None
+
+    def __post_init__(self):
+        self.show_back_button = False
+        self.title = "Switch to Electrum?"
+        self.is_bottom_list: bool = True
+
+        super().__post_init__()
+
+        self.components.append(TextArea(
+            text="It appears this is an Electrum-style seed, switch it to be so? Some functions are not supported for Electrum seeds",
```

Comment:
I wouldn't say "It appears". It IS an Electrum seed, as far as we can tell. "switch" still bugs me.

This may be too extreme, but I'm slightly more inclined to a more drastic presentation:

<img width="332" alt="Screenshot 2024-03-03 at 5 17 44 PM" src="https://github.com/SeedSigner/seedsigner/assets/934746/96441074-0256-40bd-8233-8e4d674c668e">

Reply:
Agree we can make the warning more extreme and remove the 'switch' wording.  But there is a tiny possibility it is in fact a BIP-39 seed (1 in 2^24), which is why I put a confirmation screen.  more here:
https://github.com/SeedSigner/seedsigner/pull/513#issuecomment-1976732604